master/buildbot/newsfragments/.bugfix

```diff
@@ -0,0 +1,10 @@
+When running buildbot with python3, the following API call: 
```

Comment:
the news fragment should be a single sentence. you can point to :issue:`3418` for people who would like more details.